### PR TITLE
Batch writes tutorial

### PR DESCRIPTION
**Pull request by neelp-git**

### CODE REVIEW

File: notebooks/java/batch_ops.ipynb [hunk 1]
```diff
@@ -0,0 +1,1600 @@
...
+    "printRecords()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## General Batch Operate\n",
+    "In the general form of the batch operation:\n",
+    "- A `BatchRecord is specified using a key and the specific operation details.\n",
```
Comment (reugn) [inline notebooks/java/batch_ops.ipynb:804]: Missing backtick

File: notebooks/java/batch_ops.ipynb [hunk 2]
```diff
@@ -0,0 +1,1600 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Add second namespace and restart database\n",
+    "Open a terminal tab by selecting File->Open from the notebook menu, and then New->Terminal. Run the `add-namespace.sh` script to add a namespace `test2` to the config and restart the server.\n",
+    "\n",
+    "`\n",
+    "/home/jovyan/notebooks/java/add-namespace.sh test2\n",
```
Comment (reugn) [inline notebooks/java/batch_ops.ipynb:86]: replace `/home/jovyan` with `~`

**Review by reugn — APPROVED** [review]
Well done.

**Comment by neelp-git** [comment]
Thanks @reugn . I have to add a link to the blog post which should not require another review. Will merge after that.

**Pull request closed without merging**